budget-app/src/components/atoms/IconImg/Icon.tsx

```diff
@@ -14,7 +14,7 @@ const STIcon = styled.div<IIconProps>`
   text-align: center;
 `;
 
-const Icon: React.FC<IIconProps> = ({ icon, width, height, color, backgroundColor }) => (
+const Icon: React.FC<IIcon> = ({ icon, width, height, color, backgroundColor }) => (
```

Comment:
every time I'm using this icon component I need to specify all these props??

I want to have the ability to use the component with only icon prop
give default values so I can do that